modules/network-observability-enriched-flows.adoc

```diff
@@ -37,11 +38,29 @@ spec:
         targetHost: "ipfix-collector.ipfix.svc.cluster.local"
         targetPort: 4739
         transport: tcp or udp           <4>
-
-
+ -  type: OpenTelemetry                 <3>
+      openTelemetry:
+        targetHost: my-otelcol-collector-headless.otlp.svc
+        targetPort: 4317
+        type: grpc                      <5>
+        logs:                           <6>
+          enable: true
+        metrics:                        <7>
+          enable: true
+          prefix: netobserv
+          pushTimeInterval: 20s         <8>
+          expiryTime: 2m
+   #    fieldsMapping:                  <9>
+   #      input: source.address
+   #      output: SrcAddr
 ----
 <1> The Network Observability Operator exports all flows to the configured Kafka topic.
 <2> You can encrypt all communications to and from Kafka with SSL/TLS or mTLS. When enabled, the Kafka CA certificate must be available as a ConfigMap or a Secret, both in the namespace where the `flowlogs-pipeline` processor component is deployed (default: netobserv). It must be referenced with `spec.exporters.tls.caCert`. When using mTLS, client secrets must be available in these namespaces as well (they can be generated for instance using the AMQ Streams User Operator) and referenced with `spec.exporters.tls.userCert`.
-<3> You can export flows to IPFIX instead of or in conjunction with exporting flows to Kafka.
+<3> You can export flows to IPFIX, OpenTelemetry, and Kafka individually or concurrently. 
 <4> You have the option to specify transport. The default value is `tcp` but you can also specify `udp`.
-. After configuration, network flows data can be sent to an available output in a JSON format. For more information, see _Network flows format reference_.
+<5> The protocol of Open Telemetry connection. The available options are `http` and `grpc`.
+<6> OpenTelemetry configuration for exporting logs, which are the same as the logs created by Loki. 
+<7> OpenTelemetry configuration for exporting logs, which are the same as the metrics created by Prometheus, which are specified in the `spec.processor.metrics.includeList` of the `FlowCollector` and any custom metrics you defined using the `FlowMetrics` custom resource.
+<8> The time interval that metrics are sent to the OpenTelemetry collector.
+<9> Network Observability network flows formats get automatically renamed to an OpenTelemetry compliant format. The `fieldsMapping` specification gives you the ability to rename these OpenTelemetry formats, for example, back to the original Network Observability format. In the YAML sample, `source.address` is the OpenTelemetry format and it is being renamed to the original Network Observability format, `SrcAddr` You can see both Network Observability and OpenTelemetry formats in the "Network flows format table".
```

Comment:
The Network flows format reference will be updated to with a column for the OTEL formats alongside the NetObserv formats.

Reply:
@jpinsonneau @jotak we just need to make sure that these OTEL formats get incorporated when we regenerate the Flows format reference and API references

Reply:
Ty :)